packages/frontend/.eslintrc.js

```diff
@@ -27,10 +27,7 @@ const config = {
       }
     }
   ],
-  plugins: ['vue'],
-  rules: {
-    'no-console': 1
```

Comment:
I don't see a point to this rule if there's no other way to gain insight into the application. Sure, outputting debugging information where everyone can see it isn't ideal, but not getting _any_ information about what's going on is even worse. Additionally, I don't really see the big issue with dumping debugging info in the console as long as 1) there's not too much of it 2) it doesn't leak info that we'd like to keep private.

Another way to resolve this would be to add monitoring (sentry?) to the frontend as well and then send debugging info to sentry directly (or some other target).